test/feature/user_action_pe/modification_demarche_test.dart

```diff
@@ -2,50 +2,105 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:pass_emploi_app/features/user_action_pe/list/user_action_pe_list_actions.dart';
 import 'package:pass_emploi_app/features/user_action_pe/list/user_action_pe_list_state.dart';
 import 'package:pass_emploi_app/models/user_action_pe.dart';
+import 'package:pass_emploi_app/repositories/modify_demarche_repository.dart';
 
+import '../../doubles/dummies.dart';
 import '../../doubles/fixtures.dart';
 import '../../utils/test_setup.dart';
 
 void main() {
-  test("when modifying a demarche, it should update list state", () async {
```

Comment:
Pareil: Pour rester homogène dans les noms je te propose `on successful userActionPE update, userActionsPE list is updated with modified userActionPE`
Et si on veux changer `UserActionPE` à `Demarche` on peut faire un ticket tech pour changer les noms partout 🙂